pkg/helm/helm.go

```diff
@@ -76,6 +78,14 @@ func NewCommands(k0sVars *config.CfgVars) *Commands {
 }
 
 func (hc *Commands) getActionCfg(namespace string) (*action.Configuration, error) {
+	// Construct new helm env so we get the retrying roundtripper etc. setup
+	// See https://github.com/helm/helm/pull/11426/commits/b5378b3a5dd435e5c364ac0cfa717112ad686bd0
```

Comment:
We could also include a link to the request to make the retrying roundtripper public: https://github.com/helm/helm/issues/13052.

If that's public, we could even consider to use this in k0s's kube clients as well :thinking:
And we could add this depending whether we use etcd or not.